File: lua/1116.lua
local n = io.read("*n")

for i = 1, n do
    local x = io.read("*n")
    local y = io.read("*n")

    if y == 0 then
        print("divisao impossivel")
    else
        print(string.format("%.1f", x / y))
    end
end

File: go/1116.go
package main

import "fmt"

func main() {
  var n int
  var x, y float64
  fmt.Scanf("%d", &n)
  for i := 0; i < n; i++ {
    fmt.Scanf("%f", &x)
    fmt.Scanf("%f", &y)

    if y == 0 {
      fmt.Println("divisao impossivel")
      continue
    }

    fmt.Printf("%.1f\n", x / y)
  }
}
